src/mon/PGMap.cc

```diff
@@ -1883,7 +1883,6 @@ void PGMap::dump_pool_stats(const OSDMap &osd_map, stringstream *ss,
     tbl.define_column("NAME", TextTable::LEFT, TextTable::LEFT);
     tbl.define_column("ID", TextTable::LEFT, TextTable::LEFT);
     if (verbose) {
-      tbl.define_column("CATEGORY", TextTable::LEFT, TextTable::LEFT);
       tbl.define_column("QUOTA OBJECTS", TextTable::LEFT, TextTable::LEFT);
```

Comment:
ahh, should also remove the test in `test_mon_misc()` in `qa/workunits/cephtool/test.sh`, line 629

```
/home/jenkins-build/build/workspace/ceph-pull-requests/qa/workunits/cephtool/test.sh:629: test_mon_misc:  grep CATEGORY /tmp/cephtool.8LJ/test_invalid.oCp
```

see https://jenkins.ceph.com/job/ceph-pull-requests/16254/consoleFull